docs/requests/topics/client.md

```diff
@@ -0,0 +1,10 @@
+Pass your certificate, private key, and root CA certificate to `requests.get()` (or its respective request method) to authenticate your request over TLS.
+
+```python
+result = requests.get(
+    'https://{{ server_name }}:{{ server_port }}',
+    cert=('{{ client_cert }}', '{{ client_key }}'),
+    verify='{{ ca_cert }}')
+
```

Comment:
This is not the way we do it in https://github.com/smallstep/autocert/blob/master/examples/hello-mtls/py-gunicorn/client.py#L72-L73
But I suppose it will be also ok.

Reply:
@maraino This doc is actually specifically for [requests](https://2.python-requests.org/en/master/), a popular python http client, but not part of the stdlib. I think I should probably write a separate doc for stdlib `http` or `urllib` at some point, which would look more like your gunicorn example.

Reply:
sounds good